packages/table/src/presenters/TablePresenter.js

```diff
@@ -223,11 +225,37 @@ const TablePresenter = ({
   return (
     <ThemeContext.Consumer>
       {({ resolvedRoles, metadata }) => {
-        const styles = stylesheet(
+        const tablePresenterStyles = stylesheet(
```

Comment:
to prevent us from having to do the switch in multiple places like on L233, we should pass the `stylesheet` prop to the stylesheet function, and do the check from within the function.

https://github.com/Autodesk/hig/blob/development/packages/tree-view/src/presenters/stylesheet.js#L9
https://github.com/Autodesk/hig/blob/development/packages/tree-view/src/presenters/stylesheet.js#L272

Reply:
cool, change to do the check within the stylesheet instead.